string name
bool independentFromPlanner
bool requiresExecutionSteps
string type
---

